README.md

```diff
@@ -125,9 +125,6 @@ discussions. To subscribe you can send an email with 'subscribe' in the subject
 to <nmstate-devel-join@lists.fedorahosted.org> or visit the
 [mailing list page][mailing_list].
 
-Development planning (sprints and progress reporting) happens in
-([Jira](https://nmstate.atlassian.net)). Access requires login.
```

Comment:
Thanks for removing this, but it would be nice to add that we are now tracking the sprints in Github projects. Please, see https://github.com/nmstate/nmstate/projects